README.md

```diff
@@ -311,12 +313,15 @@ Sentry.
 - **context:** Optional dictionary. See "context" note below.
 
 ### ProxyServer
-- **routes:** Required list of one or more ProxyRoute dictionaries
+- **routes:** Optional list of one or more ProxyRoute dictionaries.
```

Comment:
Maybe call out the difference between routes and streams here?

It might also be good to update the top-level features list to include TCP balancing.